gix-date/src/parse.rs

```diff
@@ -153,7 +153,8 @@ mod relative {
             "hour" => Span::new().try_hours(units),
             "day" => Span::new().try_days(units),
             "week" => Span::new().try_weeks(units),
-            // TODO months & years? YES
+            "month" => Span::new().try_months(units),
+            "year" => Span::new().try_years(units),
             // Ignore values you don't know, assume seconds then (so does git)
```

Comment:
Although it's not specific to the changes here, it occurs to me that I'm not confident I understand this comment. I don't think it's describing the behavior of this function, which does not automatically cause text like "7 blahs" to be interpreted as "7 seconds". Maybe this comment should be somewhere else? Or, is it really an accurate description even of how this function is currently used?